### ISSUE TITLE
Pr3: en `poner` podrías no usar recursión...

### ISSUE THREAD

**Issue by fidel-ml** [aguirrediego32/EstrD22s1#20]
Ya sé que es una práctica de recursión... Pero siempre es mejor usar las herramientas más simples disponibles. ;)

**Comment by aguirrediego32** [comment]
Ok, en ese caso entiendo que es mejor usar el constructor siempre que se pueda.

**Comment by fidel-ml** [comment]
Pero ahora sacaste el llamado recursivo, pero dejaste el análisis de casos que viene con la recursión. Básicamente, estás haciendo un PM para desarmar e inmediatamente volverlo a armar. Una forma sofisticada de miedo al tipo algebraico... :-D

**Comment by aguirrediego32** [comment]
Cierto! no tiene sentido la primer ecuación en este caso. Subo la corrección.

**Comment by fidel-ml** [comment]
Ojo que tenés paréntesis alrededor de algo que está solo...
